Gemfile.lock.release

```diff
@@ -529,8 +535,10 @@ GEM
       ms_rest_azure (~> 0.12.0)
     base64 (0.1.1)
     bcrypt (3.1.20)
-    bcrypt_pbkdf (1.1.0)
-    bigdecimal (3.1.6)
+    bcrypt_pbkdf (1.1.1)
+    bcrypt_pbkdf (1.1.1-arm64-darwin)
+    bcrypt_pbkdf (1.1.1-x86_64-darwin)
```

Comment:
Is this accurate?  Do we want darwin packages?

Reply:
The way the Gemfile.lock works (in newer bundlers) is that they add _every_ binary package type for all possible targets.  So, since we run the app on Macs in dev, it's included.  Nokogiri is a more thorough example:

```
    nokogiri (1.16.5)
      mini_portile2 (~> 2.8.2)
      racc (~> 1.4)
    nokogiri (1.16.5-aarch64-linux)
      racc (~> 1.4)
    nokogiri (1.16.5-arm-linux)
      racc (~> 1.4)
    nokogiri (1.16.5-arm64-darwin)
      racc (~> 1.4)
    nokogiri (1.16.5-x86-linux)
      racc (~> 1.4)
    nokogiri (1.16.5-x86_64-darwin)
      racc (~> 1.4)
    nokogiri (1.16.5-x86_64-linux)
      racc (~> 1.4)
```

Reply:
darwin strikes back

Reply:
I actually noticed this too on my first pass, and my bigger question is why the bcrypt_pbkdf gem doesn't have a linux pre-built version, and only has mac and windows 🤷 

```
# From rubygems.org:
1.1.1 - May 20, 2024 (25.5 KB)
1.1.1 - May 20, 2024 x64-mingw32 (61.5 KB)
1.1.1 - May 20, 2024 x86-mingw32 (111 KB)
1.1.1 - May 20, 2024 x86_64-darwin (94 KB)
1.1.1 - May 20, 2024 x64-mingw-ucrt (81 KB)
1.1.1 - May 20, 2024 arm64-darwin (98.5 KB) 
```